src/main/java/frc/robot/subsystems/Indications.java

```diff
@@ -52,28 +54,61 @@ public void flashing(LEDSubStrip section, Color color, int interval) {
         section.setLED(i, color);
       } else {
         // sets the LEDs off
-        section.setRGB(i, 0, 0, 0);
+        section.setLED(i, Color.kBlack);
       }
     }
   }
 
-  public void runway(LEDSubStrip section, Color color, int interval) {
+  public void runaway(LEDSubStrip section, Color color, int interval) {
     // finds many LEDS are in the section
     int sectionSize = section.getLength();
 
     for (int i = 0; i <= section.getLength(); i++) {
       // finds where is the sequence the "chasing" LED is
-      if ((i % sectionSize) == interval) {
-        armLEDsBuffer.setLED(i, color);
+      if (i % sectionSize >= (interval + sectionSize) / kIndications.maxTick
+          && (i % sectionSize) + 1 < (interval + sectionSize) / kIndications.maxTick) {
+        section.setLED(i, color);
       } else {
-        armLEDsBuffer.setRGB(i, 0, 0, 0);
+        section.setRGB(i, 0, 0, 0);
```

Comment:
Why not also use `section.setLED()` here too?
```suggestion
        section.setLED(i, color.Black);
```